packages/events-controller/lib/EventsController.js

```diff
@@ -0,0 +1,26 @@
+const EventsRepository = require('./EventsRepository');
+/**
+ * @template Data
+ * @typedef {object} IEvent
+ * @prop {Date} timestamp
+ * @prop {Data} data
+ */
+
+class EventsController {
+    static createEvents(req, res) {
+        try {
+            const {events} = req.body;
+            for (const event of events) {
+                EventsRepository.create(event);
```

Comment:
I think you can just emit the event here ☺️

Reply:
Ah so you mean we don't need to do the repository but directly use the code here ?

Reply:
I think so - the idea of a repository is to handle the persisting of data to a db - I don't think we need that here!

Reply:
Fair, i originally had just the controller but then got influenced by repository 😂 Cleaned up now, thanks for quick look

Reply:
It's all good!